aspnetcore/tutorials/build-your-first-razor-components-app.md

```diff
@@ -20,27 +20,26 @@ For an experience using ASP.NET Core Razor Components (*recommended*):
 
 * Follow the guidance in <xref:razor-components/get-started> to create a Razor Components-based project.
 * Name the project `RazorComponents`.
-* A multi-project solution is created from the Razor Components template. The Razor Components project is generated as *RazorComponents.App*.
 
 For an experience using Blazor:
 
 * Follow the guidance in <xref:spa/blazor/get-started> to create a Blazor-based project.
 * Name the project `Blazor`.
-* A single-project solution is created from the Blazor template.
+* Components continue to use the Razor class file extension (*.cshtml*) and are found in the *Pages* folder.
```

Comment:
This indicative doesn't match the previous two imperative

Reply:
> Components continue to use the Razor class file extension (\*.cshtml\*) and are found in the \*Pages\* folder.

There's no need for it here. It appears in the text just a little bit down from here. I'm striking it.